packages/web3-core-helpers/types/index.d.ts

```diff
@@ -81,16 +81,16 @@ export class errors {
     static TransactionOutOfGasError(receipt: object): TransactionError
 }
 
-export class IpcProviderBase {
-    constructor(path: string, net: net.Server);
+export class WebsocketProviderBase {
```

Comment:
I've just changed the order during copy them back to the old place but if the order is important to you will we definitely revert it.

Reply:
The diff in this PR is clearer / not mangled.